### DISCUSSION TITLE
Any way to reduce bundle size?

### DISCUSSION THREAD

**Discussion by bb4rn** [add2cal/add-to-calendar-button#366] [Q&A] [answered]
Hi,

is there any way to only import the functionality I need, in order to reduce bundle size?

I’m using a single custom button with my own css, for only one type of calendar, so no dropdown or any effect.

(I’m using Parcel with tree shaking enabled.)

Thanks!

**Comment by jekuer** [comment, marked as answer]
sorry for the late response.
We are thinking of offering different options to import the script. default (all styles) or one specific style. this should reduce bundle size, if you know what you want.
Offering css as css again would be better, but contradicts with our goal to minimize the setup time.
Will take some more time, since this is rather further down the roadmap, but on it.
